server/client.go

```diff
@@ -3755,9 +3814,15 @@ func (c *client) processMsgResults(acc *Account, r *SublistResult, msg, deliver,
 			// Assume delivery subject is normal subject to this point.
 			dsubj = subj
 			// Check for stream import mapped subs. These apply to local subs only.
-			if sub.im != nil && sub.im.prefix != "" {
-				dsubj = append(_dsubj[:0], sub.im.prefix...)
-				dsubj = append(dsubj, subj...)
+			if sub.im != nil {
```

Comment:
Looks the same than the one just above. Could maybe make it a function (one inside this function if we want to make use of _dsubj), but it's ok to duplicate..

Reply:
Yep saw that and just decided to dupl like before.